libs/designsystem/card/src/card.component.spec.ts

```diff
@@ -2,12 +2,16 @@ import { TestHelper } from '@kirbydesign/designsystem/testing';
 import { createHostFactory, SpectatorHost } from '@ngneat/spectator';
 
 import { CardComponent } from './card.component';
+import { CardHeaderComponent } from './public_api';
```

Comment:
Consider aligning import with line above:
```suggestion
import { CardHeaderComponent } from './card-header/card-header.component';
```